www/docs/deploying.md

```diff
@@ -1,3 +1,62 @@
 # Deploying a Skip service
 
+The process of setting up and deploying a Skip service is mostly similar to that of other backend systems (databases, caches, etc.) but there are some important nuances to be aware of.
+
+This page walks through those details, or you can refer to the examples [setup](https://github.com/SkipLabs/skip/tree/main/examples/hackernews) for a representative and complete practical configuration based on standard tooling including Docker, HAProxy, Flask, and React.
+
 ## Overview
+
+Skip reactive services define an HTTP interface for reading, writing, and subscribing to data.
+In production use cases for web applications, this should be deployed behind a reverse proxy such as Nginx, HAProxy, or Traefik which is responsible for traffic encryption, load balancing, HTTP tunnelling, and dispatching requests to backend services.
+Throughout this page we will show HAProxy configuration in examples, but it should be directly applicable to other proxies.
+
+Skip is agnostic to your choice of reverse proxy, but it is important that it is configured with HTTP/2 for request/response multiplexing, to avoid browser limitations on concurrent server-sent event connections, and to ensure that TLS is used.
+
+### Running your service
+
+We recommend using Docker Compose to package and run your reactive service.
```

Comment:
Do we want to say this as strongly as "We recommend using", or would something like "One possibility is to use"?